lstchain/scripts/lstchain_cherenkov_transparency.py

```diff
@@ -174,7 +174,7 @@ def main():
                 dummy = np.zeros(num_subruns)
                 dummy2 = np.zeros(num_subruns)
                 for jj, nn, nn2 in zip(
-                        a.root.dl1datacheck.pedestals.col('subrun_index'),
+                        np.arange(0,num_subruns),
```

Comment:
This will not work as intended. In case pedestals (or flat-field) table is missing, we want to fill the values for the existing subruns in the right subrun index. But now jj is no longer the subrun index.